packages/flutter_tools/lib/src/application_package.dart

```diff
@@ -116,9 +124,12 @@ class AndroidApk extends ApplicationPackage {
     for (xml.XmlElement category in document.findAllElements('category')) {
       if (category.getAttribute('android:name') == 'android.intent.category.LAUNCHER') {
```

Comment:
According to `https://stackoverflow.com/questions/25219551/what-is-the-meaning-of-android-intent-action-main` we should be looking for `android.intent.action.MAIN` instead.

Reply:
That was intended as a question. `android.intent.action.ASK`